crates/cairo-profiler/src/main.rs

```diff
@@ -36,15 +38,20 @@ struct Cli {
     /// Specify maximum depth of function tree in function level profiling.
     /// The is applied per entrypoint - each entrypoint function tree is treated separately.
     /// Keep in mind recursive functions are also taken into account even though they are later
-    /// aggregated to one function call.
+    /// aggregated to a single function call.
     #[arg(long, default_value_t = 100)]
     max_function_trace_depth: usize,
+
+    /// Split non-inlined generic functions based on the type they were monomorphised with.
```

Comment:
Why not-inlined is mentioned here? How it is relevant to this flag.

Reply:
https://github.com/software-mansion/cairo-profiler/pull/63#discussion_r1609801928